tests/integration/sync_test.go

```diff
@@ -852,3 +770,37 @@ func Test_Sync_RateLimitingPlugin(t *testing.T) {
 		})
 	}
 }
+
+func Test_Sync_SkipCACert(t *testing.T) {
+	// setup stage
+	client, err := getTestClient()
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	tests := []struct {
+		name          string
+		kongFile      string
+		expectedState utils.KongRawState
+	}{
+		{
+			name:     "syncing with --skip-ca-certificates should ignore CA certs",
+			kongFile: "testdata/sync/008-skip-ca-cert/kong.yaml",
+			expectedState: utils.KongRawState{
+				Services:       svc1_207,
+				CACertificates: []*kong.CACertificate{},
+			},
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			kong.RunWhenKong(t, ">=2.7.0")
```

Comment:
Is there a reason that this is gated at 2.7? CA certificates were added [much earlier](https://github.com/Kong/kong/blob/master/CHANGELOG.md#core-42) and I'm not aware of any API difference that'd affect the way this flag operates since.

Reply:
I did it mostly to avoid having N tests for this, due to schema differences on various version. If you think that's needed/good to have, I'll split the tests per-version

Reply:
What are the differences? If the flag functions the same either way and it's just a matter of the input/output objects not being identical across all versions, fine to leave the gate in place as-is, just add a comment to the effect of "ca_certificates first appeared in 1.3, but we limit to 2.7+ here because the schema changed and the entities aren't the same across all versions, even though the skip functionality works the same."

Reply:
Exactly. Done.